packages/core/addon/mirage/fixtures/reports.js

```diff
@@ -26,7 +26,8 @@ export default [
                 dimensions: [
                   { name: 'Property 1', values: { property: '114' } },
                   { name: 'Property 2', values: { property: '100001' } },
-                  { name: 'Property 3', values: { property: '100002' } }
+                  { name: 'Property 3', values: { property: '100002' } },
+                  { name: 'Property 4', values: { property: '101272' } }
```

Comment:
Why do we need to add an additional property?

Reply:
When running the tests I found that a Property 4 was showing up regardless of if I had one defined. It showed up with the value of 101272 so I added it and everything worked as expected with it defined properly.